ui/components/Page.tsx

```diff
@@ -16,13 +16,13 @@ export type PageProps = {
   children?: any;
   title?: string;
   breadcrumbs?: { page: PageRoute; query?: any }[];
+  topRight?: any;
```

Comment:
Assuming this is an element:

```suggestion
  topRight?: JSX.Element;
```